release/models/interfaces/openconfig-if-ip.yang

```diff
@@ -296,6 +302,21 @@ module openconfig-if-ip {
           Internet Protocol (IP)";
       }
 
+      leaf in-multicast-pkts {
```

Comment:
That's fair, but most (all?) of these counters are already duplicates of interface counters.  But they have different context because they are in the IP container.  A multicast Ethernet packet may not be the same counter as a multicast IP packet.

I will try refactoring the groupings to see if I can make a sensible reusable grouping for common Interface and IP counters.  But the descriptions will be a bit compromised I think?   (references to IP MIB don't make sense for a leaf defined at the interface level that is reused for Ethernet and IP).  I think it is ok/doesn't matter that much.  For the examples I can think of this is ok and is perhaps only a matter of how much one prefers an elegant model versus precision of description.

Reply:
Per our quick chat, if it's just the references I do not think this is a problem -- but if it's fundamental description differences then that seems more problematic.

Reply:
It's just the RFC references.  I merged the RFC references, probably it is straightforward.  You can decide.